Content.Shared/GPS/Components/HandheldGPSComponent.cs

```diff
@@ -0,0 +1,16 @@
+
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.GPS.Components
+{
+    [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
```

Comment:
Add `AutoGenerateComponentPause` so it automatically deals with pauses for the update time stuff. I guess it wont really change much because you won't be moving but its probably a good idea anyway 😆